testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -965,6 +965,31 @@ def test_first_residue(self, universe):
         assert len(universe.residues[0].atoms) == 19
 
 
+class TestPDBVaryingOccTmp:
+    @staticmethod
+    @pytest.fixture(scope='class')
+    def u():
+        return mda.Universe(PDB_varying)
+
+    def test_ts_data_keys(self, u):
+        ts = u.trajectory.ts
+
+        assert 'occupancy' in ts.data
+        assert 'tempfactor' in ts.data
+
+    def test_varying_occupancy(self, u):
```

Comment:
Could you:

a) parametrize these two (seems easy enough with two keywords)
b) maybe do a more traditional trajectory iteration and rewind just to check everything works as expected?